metomi/rose/config_processors/fileinstall.py

```diff
@@ -865,7 +866,7 @@ def parse(self, loc, conf_tree):
                 if handler is None:
                     handler = self.guess_handler(loc)
                 if handler is None:
-                    raise ValueError(loc.name)
+                    raise ValueError(f"don't know how to process {loc.name}")
```

Comment:
[minor] maybe a slightly cryptic error message for users to digest:

```suggestion
                    raise ValueError(f"Rose doesn't know how to process {loc.name}")
```